### PR TITLE
some commtrack case fixes

### PR DESCRIPTION
**Pull request by snopoke**
@dannyroberts

can review by commit

### CODE REVIEW

File: corehq/ex-submodules/casexml/apps/case/signals.py [hunk 1]
```diff
@@ -1,11 +1,17 @@
 from django.dispatch.dispatcher import Signal
+from casexml.apps.stock.models import StockReport, StockTransaction
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/case/signals.py]: F401 'StockReport' imported but unused

**Comment by czue** [comment]
kicked build

**Comment by czue** [comment]
failed again. possibly related?
```
======================================================================
ERROR: test_undo_data2 (custom.ewsghana.tests.handlers.undo.TestUndo)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/custom/ewsghana/tests/handlers/undo.py", line 75, in test_undo_data2
    self.run_script(a)
  File "/home/travis/build/dimagi/commcare-hq/custom/logistics/tests/test_script.py", line 47, in run_script
    self.assertEqual(msg.text, unicode(command['text']))
AttributeError: 'NoneType' object has no attribute 'text'
----------------------------------------------------------------------
```

**Comment by snopoke** [comment]
:white_check_mark:

**Pull request merged**